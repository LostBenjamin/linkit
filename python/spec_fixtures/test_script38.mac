
[app1 10]

    <NUM-LEDS>
    get-random-number

    set-position 
    {{
        red
        set-breathe
    }}

[app2 11]

    <NUM-LEDS>
    get-random-number

    set-position {{
        blue
        set-blink
    }}

