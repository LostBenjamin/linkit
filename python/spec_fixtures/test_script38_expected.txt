10:set:90:rng:pos:red:bre:rst
11:set:90:rng:pos:blu:bli:rst

